lib/msnodesqlv8.js

```diff
@@ -502,7 +536,9 @@ class Request extends base.Request {
 
             let exi = row[columns[idx].name]
             if (exi != null) {
-              if (exi instanceof Array) {
+              if (typeof exi === 'string' && typeof data === 'string' && columns[idx].type === 'text') {
```

Comment:
This breaks the current behavior when recordset contain multiple columns with same name. I have started discussion about this here - https://github.com/TimelordUK/node-sqlserver-v8/issues/37